study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/BrainPediatricsUpdates.java

```diff
@@ -106,15 +106,23 @@ public void run(Handle handle) {
         helper.update18nActivitySubtitle(activityDto.getActivityId(), newSubtitle);
         LOG.info("updated variables for release styling changes");
 
+        //update workflow expr for ABOUTYOU TO POSTCONSENT
+        long wfTransitionId = helper.findWorkflowTransitionId(studyId, "ABOUTYOU", "POSTCONSENT");
+        String newExpr = "user.studies[\"cmi-brain\"].forms[\"CONSENT\"].isStatus(\"COMPLETE\") && "
+                + "(user.studies[\"cmi-brain\"].forms[\"POSTCONSENT\"].isStatus(\"CREATED\") "
+                + "|| user.studies[\"cmi-brain\"].forms[\"POSTCONSENT\"].isStatus(\"IN_PROGRESS\"))";
+        helper.updateWorkflowExpressionTextById(wfTransitionId, newExpr);
+
```

Comment:
changes to handle edge cases fo existing users who signed up but didn't complete self consent